src/avt/Queries/Queries/avtXRayImageQuery.C

```diff
@@ -151,7 +189,7 @@ avtXRayImageQuery::avtXRayImageQuery():
     backgroundIntensities = NULL;
     nBackgroundIntensities = 0;
     debugRay = -1;
-    familyFiles = false;
+    filenameScheme = NONE;
```

Comment:
our default before was to have familying turned off.

Reply:
Oh, ok, I am thinking of VisIt's save window options. Just curious but how much of the logic you've done here is *general* and useful to apply in other places where we *family* files?

Reply:
I have tried to keep the familying logic intact and simply standardize the naming conventions. The code block in the execute function with the while loop in it isn't very general. That's where the meat of this is. Perhaps it could be generalized. I tried to change it as little as possible while cleaning things up and making them consistent.

Reply:
To answer your question specifically, almost _none_ of the logic I've done here is general and useful to apply in other places. Most of the logic that _is_ here was not written by me. This logic is also not generalized, and is quite specific to the x ray image query. In the future, perhaps we could investigate ways to generalize this. I think the work I've done is a big _step_ towards making the familying more general, as I've tried to remove as many idiosyncrasies and specific behaviors as possible. But everything is still quite specific to the x ray image query.